modules/evm/src/lib.rs

```diff
@@ -84,10 +117,12 @@ pub struct GenesisAccount<Balance, Index> {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as EVM {
-		AccountNonces get(fn account_nonces): map hasher(twox_64_concat) H160 => T::Index;
-		AccountCodes get(fn account_codes): map hasher(twox_64_concat) H160 => Vec<u8>;
+		Accounts get(fn accounts): map hasher(twox_64_concat) H160 => Option<AccountInfo<T::Index>>;
 		AccountStorages get(fn account_storages):
 			double_map hasher(twox_64_concat) H160, hasher(blake2_128_concat) H256 => H256;
+
+		Codes get(fn codes): map hasher(blake2_128_concat) H256 => Vec<u8>;
```

Comment:
```suggestion
		Codes get(fn codes): map hasher(identity) H256 => Vec<u8>;
```

The key should already be a secured hash